app/models/user.js

```diff
@@ -23,7 +23,9 @@ const UserSchema = new Schema({
   facebook: {},
   twitter: {},
   github: {},
-  google: {}
+  google: {},
+  active: { type: Boolean, required: true, default: false },
+  temporaryToken: { type: String, default: true }
```

Comment:
This field shouldn't be in the model anymore. @ddouglasz

Reply:
Okay I will take that off now. Thank you.